src/Integration/Suppression/SonarQubeIssuesProvider.cs

```diff
@@ -156,11 +171,12 @@ private async Task SynchronizeSuppressedIssuesAsync()
                 var moduleKeyToRelativePathToRoot = (await this.sonarQubeService.GetAllModulesAsync(sonarQubeProjectKey,
                         cancellationTokenSource.Token))
                     .ToDictionary(x => x.Key, x => x.RelativePathToRoot ?? string.Empty);
+                this.hasModules = moduleKeyToRelativePathToRoot.Keys.Count == 1;
```

Comment:
Isn't it the case that if there is only one module then that must be the root module -> _hasModules = false_?

If the code as-is is incorrect then there must a gap in the test coverage.